Cargo.toml

```
@@ -93,4 +93,4 @@ arrow-select = { version = "54.2.0", path = "./arrow-select" }
 arrow-string = { version = "54.2.0", path = "./arrow-string" }
 parquet = { version = "54.2.0", path = "./parquet", default-features = false }
 
-chrono = { version = "0.4.34", default-features = false, features = ["clock"] }
+chrono = { version = "< 0.4.40", default-features = false, features = ["clock"] }
```

```suggestion
chrono = { version = "0.4.34, < 0.4.40", default-features = false, features = ["clock"] }
```
Or whatever the syntax is

---

`< 0.4.40` should them all, I guess.

---

I think that would permit `0.4.32` which would then fail to build

---

Oh, you are right.

---

Fixed in https://github.com/apache/arrow-rs/pull/7210/commits/96674289a7c8ca8508dd67aafb17334f97b47d84